diag_manager/fms_diag_object.F90

```diff
@@ -426,9 +427,16 @@ subroutine fms_diag_send_complete(this, time_step)
 #else
   class(fmsDiagFileContainer_type), pointer :: diag_file !< Pointer to this%FMS_diag_files(i) (for convenience)
 
+  logical :: file_is_opened !< True if the file was opened in this time_step
```

Comment:
This variable should be named something like `file_opened_this_time_step` so it's more descriptive.